src/afl-tmin.c

```diff
@@ -1055,6 +1060,24 @@ int main(int argc, char **argv_orig, char **envp) {
         read_bitmap(optarg, mask_bitmap, map_size);
         break;
 
+      case 'l':
+        if (del_limit_given) { FATAL("Multiple -l options not supported"); }
+        del_limit_given = 1;
+
+        if (!optarg) { FATAL("Wrong usage of -l"); }
+
+        if (optarg[0] == '-') { FATAL("Dangerously low value of -l"); }
+
+        del_len_limit = atoi(optarg);
+
+        if (del_len_limit < 1 || del_len_limit >= TMIN_MAX_FILE) {
```

Comment:
so 0 would be a valid value?

Reply:
The existing `del_len` decreases in this manner: (the largest power of 2 less than the file size), 16, 8, 4, 2, 1, ... . So a value of 0 seems to be invalid.